File: ETI06F1/source.c
/* Autor: Bartłomiej Romanek         */
/* Tytuł: SPOJ. Kod zadania: ETI06F1 */
/* Licencja: Licencja MIT            */
/* Data modyfikacji: 01.07.2016      */

#include <stdio.h>

double R, D, P, PI = 3.141592654;

int main() {
	scanf("%lf %lf", &R, &D); 
	P = ((R*R)-((D*D)/4))*PI;
	printf("%0.2lf", P);
	return 0;
}

/* Opis kodu: W tym zadaniu najważniejsze jest określenie wzoru na pole nowo powstałego koła. */

File: SUMA/source.js
/* Autor: Bartłomiej Romanek      */
/* Tytuł: SPOJ. Kod zadania: SUMA */
/* Licencja: Licencja MIT         */
/* Data modyfikacji: 02.07.2016   */

var sum = 0;
var n = readline();
putstr(sum += parseInt(n));
while ((n = readline()) !== null) {
	putstr("\n" + (sum += parseInt(n)));
}

// Opis kodu: W tym zadaniu najważniejsze jest zwrócenie uwagi na znaki kończące linie tesktu - nie są dopuszczalne wolne  
// linie na początku oraz na końcu serii wyników.
